packages/atclient/src/monitor.c

```diff
@@ -150,10 +145,8 @@ int atclient_monitor_read(atclient *monitor_conn, atclient *atclient, atclient_m
     goto exit;
   }
 
-  int i = 0;
-  while (buffer[i] != ':') {
-    i++;
-  }
```

Comment:
this was a hack to align the monitor messages, which should be resolved by the parse_message changes below